Doc/library/importlib.rst

```diff
@@ -815,7 +815,7 @@ ABC hierarchy::
 
     .. versionadded:: 3.9
 
-    .. abstractmethod:: name()
+    .. property abstractmethod:: name()
```

Comment:
Is `property abstractmethod` the right syntax?  I don't see that anywhere else in the docs.

Reply:
Indeed, this syntax seems not to be supported as the block doesn't render anything:

![image](https://user-images.githubusercontent.com/308610/139588573-b4506a90-2981-4ec4-a700-93911a088ad2.png)

Reply:
I found [one other example](https://github.com/python/cpython/blame/086b5c6ce16b524629428b4fd5c9213930bfca47/Doc/library/numbers.rst#L33-L35) of an abstract property. It's documented as an attribute with "Abstract" in the text. Let's follow that pattern until there's something better.

Reply:
Fixed in f89e2c037a.